addons/website_cf_turnstile/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    module_website_cf_turnstile = fields.Boolean(string='Use CloudFlare Turnstile')
```

Comment:
Not really useful. This checkbox is by definition checked as it is the current module.

Reply:
Just don't create this field in stable.
(and it avoids an upgrade script to move the field)

Reply:
Je peux le remplacer par un boolean compute si ICP set si tu preferes. Ainsi on a bien la checkbox dans l'uX (ce qu'on veut :/) pour ne pas faire 'différent des autres' et on ajoutera le boolean module_xx en master. Avec une fonction inverse quand tu décoches pour reset icp au lieu de désinstaller/désactiver le module sans perdre les clés ... 

+ de code, pour moins bien... est ce vraiment ça qu'on veut ?

![image](https://user-images.githubusercontent.com/4848797/227255538-95c5956b-312a-4351-a988-06803f53e129.png)